unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -90,13 +87,15 @@ public int numBytes(final byte b) {
    */
   public int length() {
```

Comment:
Yea I think it's best to not have it named length because it is a little bit ambiguous here. It could mean length of the string in terms of characters, or length in terms of number of bytes.